deploy/kubernetes-1.13/pmem-app.yaml

```diff
@@ -10,8 +10,6 @@ spec:
       volumeMounts:
       - mountPath: "/data"
         name: my-csi-volume
-  nodeSelector:
-    storage: pmem
```

Comment:
@pohly I think, we still not ready for this. If I remember correctly, the issue is when the cluster has schedulable nodes without "storage: pmem" label, that case Kubernetes might schedule the application pod on those nodes, even if it requires pmem volume.

Reply:
Have you considered that we currently don't use delayed binding? If we did, what you said can indeed happen. But if we always create volumes first and Kubernetes honors the topology (both should be the case), then dropping the node selector is fine.

Reply:
yes, if we ignore delayed binding. This should work.